openpype/tools/push_to_project/control_integrate.py

```diff
@@ -970,7 +970,7 @@ def make_sure_version_exists(self):
             last_version_doc = get_last_version_by_subset_id(
                 project_name, subset_id
             )
-            version = 1
+            version = get_versioning_start()
             if last_version_doc:
                 version += int(last_version_doc["name"])
 
```

Comment:
Not related to this PR but I'm pretty sure this `if version is None:` shouldn't be here to begin with. It's already handled at line 939 right?
Line 969 - 976 can be removed.

Reply:
Yeah, good spot